WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+final class SiteInformationWizardContent: UIViewController {
+    fileprivate enum Rows: Int, CaseIterable {
+        case title = 0
+        case tagline = 1
+
+        static func count() -> Int {
+            return allCases.count
+        }
+
+        func matches(_ row: Int) -> Bool {
+            return row == self.rawValue
+        }
+    }
+
+    private let completion: (SiteInformation) -> Void
+
+    @IBOutlet weak var table: UITableView!
+    @IBOutlet weak var nextStep: UIButton!
+
+    private lazy var headerData: SiteCreationHeaderData = {
+        let title = NSLocalizedString("Basic information", comment: "Create site, step 3. Select basic information. Title")
+        let subtitle = NSLocalizedString("Tell us more about the site you are creating.", comment: "Create site, step 3. Select basic information. Subtitle")
+
+        return SiteCreationHeaderData(title: title, subtitle: subtitle)
+    }()
+
+    init(completion: @escaping (SiteInformation) -> Void) {
```

Comment:
What do you think about defining a `typealias` for this, and using it both here and on line 17?

Reply:
Yep, nice!